ArduCopter/land_detector.cpp

```diff
@@ -298,8 +298,8 @@ void Copter::update_throttle_mix()
         // autopilot controlled throttle
 
         // check for aggressive flight requests - requested roll or pitch angle below 15 degrees
-        const Vector3f angle_target = attitude_control->get_att_target_euler_cd();
-        bool large_angle_request = angle_target.xy().length() > LAND_CHECK_LARGE_ANGLE_CD;
+        const Vector3f angle_target_rad = attitude_control->get_att_target_euler_rad();
```

Comment:
we could use a const reference here too

const Vector3f& angle_target_rad = attitude_control->get_att_target_euler_rad();